src/oauth.rs

```diff
@@ -0,0 +1,88 @@
+use crate::settings;
+use crate::token::{verify_jwt_token_from_rsa, Claims};
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug)]
+pub struct Response {
+    pub email: String,
+    pub claims: Claims,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Key {
+    // kty: String,
+    pub n: String, // We only need n and e
+    pub e: String,
+    // qi: String,
+    // kid: String,
+    // alg: String,
+    // r#use: String,
+    // fxa-createdAt: String,   // This can't be named this hence serde_json
+    // won't parse this resulting in error
```

Comment:
Since we can't name this variable 'fxa-createdAt' like this due to limitation of rust(serde_json requires it to be name like this). Then how you suggest we resolve this issue?

Reply:
@jrconlin - do you have any suggestions here?

Reply:
I made a suggestion in [the prior comment](https://github.com/mozilla-services/tokenserver-rs/pull/17#discussion_r499057022) that field attributes may help. In this case, `rename` may be useful and allow the variable to be renamed when importing or exporting.